test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1

```diff
@@ -2108,7 +2108,10 @@ Describe "Import-PSSession on Restricted Session" -tags "Feature","RequireAdminO
         }
     }
 
-    It "Verifies that Import-PSSession works on a restricted session" {
+    # Blocked by https://github.com/PowerShell/PowerShell/issues/4275
+    # Need a way to created restricted endpoint based on a different endpoint other than microsoft.powershell which points
+    # to Windows PowerShell 5.1
+    It "Verifies that Import-PSSession works on a restricted session" -Pending {
```

Comment:
I just noticed that similar implicit remoting tests are disabled using special checks:
```
# Skip tests for CoreCLR for now
# Skip tests on ARM
$skipTest = $IsCoreCLR -or $env:PROCESSOR_ARCHITECTURE -eq 'ARM'

if ($skipTest) { return }
```
We should make this test be the same since remoting really does not work yet with PowerShell 6.0 based endpoints.  Or maybe we should mark all tests that aren't supported as "pending" since we no longer support FullCLR.